extendr-macros/src/wrappers.rs

```diff
@@ -471,20 +451,15 @@ fn translate_to_robj(input: &FnArg) -> syn::Result<syn::Stmt> {
 }
 
 // Generate actual argument list for the call (ie. a list of conversions).
-fn translate_actual(opts: &ExtendrOptions, input: &FnArg) -> Option<Expr> {
+fn translate_actual(input: &FnArg) -> Option<Expr> {
```

Comment:
how come this has removed the options?

Reply:
Right. Well, this is a good question.

Before, I had to write two versions based on whether we used try-from or not. But I don't have to now.
I can assume try-from.

Reply:
Do this ignore the other options like name and rng?